java/code/src/com/suse/manager/webui/services/impl/SaltSSHService.java

```diff
@@ -430,8 +437,8 @@ else if (e != null) {
             }
 
             proxyCommand.append(String.format(
-                    "/usr/bin/ssh -i %s -o StrictHostKeyChecking=no -o User=%s%s %s ",
-                    key, PROXY_SSH_PUSH_USER, stdioFwd, proxyHostname));
+                    "/usr/bin/ssh -p %s -i %s -o StrictHostKeyChecking=no -o User=%s%s %s ",
```

Comment:
Actually Java is handling Salt SSH roster data only for bootstrapping, so to make it fully working we need to modify Uyuni roster salt module also: https://github.com/uyuni-project/uyuni/blob/master/susemanager-utils/susemanager-sls/modules/roster/uyuni.py

Reply:
Thanks for the heads up, I'll look into it. I guess the way to test this to run any Salt command on a Salt minion using such a proxy... or is it for Salt-SSH minions only?

Reply:
As I see in case of using the proxy with the different port than 22 to reach any minion is not possible with Uyuni roster now, so we need to align it the same way as you did in Java side to build roster data the proper way. On testing I think reaching the salt ssh clients behind such proxies will not be possible without modifying the uyuni roster module.